### DISCUSSION TITLE
Constant error while adding new locations

### DISCUSSION THREAD

**Discussion by Gwen-the-pen** [Monika-After-Story/MonikaModDev#9396] [QA - code and content creation (technical)]
Whenever I try to add my own location (i.e. using my own backgrounds) the game always gets an exception and for some reason can **NEVER** find the file for the background. I'm SO confused and don't know what to do.

Specifically, the "floating sky" background from this google drive - https://drive.google.com/drive/u/0/folders/1fal2EX-cTrmE0GBvhLEzjTPnijmE6F3W

**Comment by Wingdinggaster656** [comment]
Did you follow the wiki guide page?
https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Custom-Backgrounds

**Reply by Gwen-the-pen** [reply]
I did - multiple time. I still got the same error
